pkg/operator/ceph/cluster/mon/config.go

```diff
@@ -32,7 +32,7 @@ import (
 	"github.com/rook/rook/pkg/operator/k8sutil"
 	"github.com/rook/rook/pkg/util/exec"
 	"github.com/rook/rook/pkg/util/sys"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
There should be no need to specify this as `v1` since the module is already called `v1`